packages/base-driver/lib/protocol/protocol.js

```diff
@@ -262,6 +262,15 @@ function buildHandler(app, method, path, spec, driver, isSessCmd) {
     let currentProtocol = extractProtocol(driver, req.params.sessionId);
 
     try {
+      // if the route accessed is deprecated, log a warning
+      if (spec.deprecated) {
+        getLogger(driver, req.params.sessionId).warn(
+          `Command '${spec.command}' has been deprecated and will be removed in a future ` +
```

Comment:
not sure if the message should be shorter. A long message might seriously increase the log size if the particuluar command is called multiple times.

Reply:
Agree.  One warning per route.  Probably just keep a `Set<string>` of routes that have had this warning emitted and check it.